include/seqan3/alignment/pairwise/edit_distance_unbanded.hpp

```diff
@@ -236,6 +264,17 @@ struct pairwise_alignment_edit_distance_unbanded
 
         add_state();
     }
+
+    pairwise_alignment_edit_distance_unbanded(
+        database_t _database,
+        query_t _query,
+        align_config_t _config,
+        traits_t /*_traits*/) :
+            pairwise_alignment_edit_distance_unbanded(std::forward<database_t>(_database),
+                                                      std::forward<query_t>(_query),
+                                                      std::forward<align_config_t>(_config))
+    {}
```

Comment:
This constructor is not needed. When someone wants to use the `traits_t` parameter (which should only when testing the code) he should specify everything.